src/utils/table/Table.tsx

```diff
@@ -6,12 +6,13 @@ import * as styles from './Table.scss';
 export const Table = ({ children, align, scrollable, className }:
       { children: any,
         align?: 'right' | 'left' | 'center',
-        scrollable?: any,
+        scrollable?: boolean | 'lg',
```

Comment:
I dont like it. You combine multiple types. Scrollable<Boolean> is fine but the other one should be in separatr property maybe.  What about size ?

Reply:
I had smth in mind with 'lg' scrollable, but when tried to write explanation it turns out overcomplicated ;) I'm removing this.